### ISSUE TITLE
Ironbug Tutorial or User Manual

### ISSUE THREAD

**Issue by PierreCrespin** [MingboPeng/Ironbug#59]
Hi Mingbo,

I am trying to use your Ironbug suite with latest honeybee and ladybug tools for a project. However, I cannot find how to assign the HVAC systems to the HB rooms and I see that the new OSM Simulation component doesn't have the HVAC System input option as the old Legacy tool one had. Is there an example script with latest LBT, a tutorial or just a user manual that I could use to understand the how the workflow is supposed to work ? Thanks in advance for your help on this matter. 
Kind regards,

Pierre

![image](https://user-images.githubusercontent.com/115151228/194298800-7b0fc875-d5fc-451b-b81e-540ea559ea31.png)

**Comment by passivstrang** [comment]
I agree with Pierre. I see the great value of Ironbug as part of the LBT ecosystem, but I can't intuitively work out how to use with without a few tutorials or a user manual.

**Comment by TfedUD** [comment]
Hi @PierreCrespin If you can post on the [LBT Forum](https://discourse.ladybug.tools/) with a simplified version of the file above I see where some of the issues are in your file; and can edit re-upload a working one and clarify a few things hopefully!

**Comment by PierreCrespin** [comment]
Hi @TfedUD , apologies for the late answer and thank you for your reply. I have now posted this query on the LBT forum. However, I couldn't include the script in the post. Is there any way I could send that over to you for review ? Thanks for your help on this.
